### ISSUE TITLE
app build issue

### ISSUE THREAD

**Issue by arindamgit** [mmig/cordova-plugin-gadgetbridge#2]
Thank you for writing and publishing this amazing plugin. Also thanks to the main Gadgetbridge team.

I am trying to use the plugin in an ionic 3 app.

When I try to run/build the project I get this

`
[cordova]  FAILURE: Build failed with an exception.
[cordova]  
[cordova]  * What went wrong:
[cordova]  Could not resolve all files for configuration ':app:debugCompileClasspath'.
[cordova]  > Could not find :gadgetbridge:.
[cordova]    Searched in the following locations:
[cordova]        file:/home/arindam/Android/Sdk/extras/m2repository//gadgetbridge//gadgetbridge-.pom
[cordova]        file:/home/arindam/Android/Sdk/extras/m2repository//gadgetbridge//gadgetbridge-.aar
[cordova]        file:/home/arindam/Android/Sdk/extras/google/m2repository//gadgetbridge//gadgetbridge-.pom
[cordova]        file:/home/arindam/Android/Sdk/extras/google/m2repository//gadgetbridge//gadgetbridge-.aar
[cordova]        file:/home/arindam/Android/Sdk/extras/android/m2repository//gadgetbridge//gadgetbridge-.pom
[cordova]        file:/home/arindam/Android/Sdk/extras/android/m2repository//gadgetbridge//gadgetbridge-.aar
[cordova]        https://jcenter.bintray.com//gadgetbridge//gadgetbridge-.pom
[cordova]        https://jcenter.bintray.com//gadgetbridge//gadgetbridge-.aar
[cordova]        https://maven.google.com//gadgetbridge//gadgetbridge-.pom
[cordova]        https://maven.google.com//gadgetbridge//gadgetbridge-.aar
[cordova]        https://repo1.maven.org/maven2//gadgetbridge//gadgetbridge-.pom
[cordova]        https://repo1.maven.org/maven2//gadgetbridge//gadgetbridge-.aar
[cordova]        https://jitpack.io//gadgetbridge//gadgetbridge-.pom
[cordova]        https://jitpack.io//gadgetbridge//gadgetbridge-.aar
[cordova]        file:/home/arindam/AppRoot/band/platforms/android/app/libs/gadgetbridge-.aar
[cordova]        file:/home/arindam/AppRoot/band/platforms/android/app/libs/gadgetbridge.aar
[cordova]    Required by:
[cordova]        project :app
`
Is there a way to resolve this

**Comment by arindamgit** [comment]
I resolved this by copying gadgetbridge.aar from 
`plugins/cordova-plugin-gadgetbridge/res/android/libs/gadgetbridge.aar`
to
`platforms/android/app/libs/`

is that the correct way of resolving this
Thanks you for your attention

**Comment by russaa** [comment]
sorry for the late reply.

as to your question:
yes, you correctly addressed the error, but really, this (manually copying the AAR file) should not have been necessary at all: the installion of the plugin should have taken care of that already (copying the AAR file to the correct location).

Could you give some more details/context?

How did you install the plugin?
e.g. explicitly via command-line or by configuration via `config.xml`/`package.json`?

Did error messages occur during installation of the plugin?

Did you try de-installing & re-installing the plugin via command-line?
i.e. something like
```
cordova plugin rm cordova-plugin-gadgetbridge
cordova plugin add https://github.com/mmig/cordova-plugin-gadgetbridge.git
```

**Comment by russaa** [comment]
OK, sorry for the confusion:

it seems the problem was the (relatively) new `cordova-android` platform 7 which has some breaking changes

-> fixed now with plugin version 0.3.3

**Comment by arindamgit** [comment]
Thank you
